main/appserver/packager/appserver-base/src/main/docs/README.txt

```diff
@@ -1,4 +1,4 @@
-Thank you for downloading GlassFish Server Open Source Edition 4.1!
```

Comment:
This file has some more reference to 4.1. However those are mostly related to release notes . So I am not changing them